homeassistant/components/bom/sensor.py

```diff
@@ -240,7 +241,7 @@ def should_update(self):
             # Never updated before, therefore an update should occur.
             return True
 
-        now = datetime.datetime.now()
+        now = dt_util.utcnow()
         update_due_at = self.last_updated + datetime.timedelta(minutes=35)
```

Comment:
Don't we need to update `last_updated` to utc time too?

Reply:
Fixed.